frontend/HomeTabs.tsx

```diff
@@ -0,0 +1,52 @@
+import { createMaterialBottomTabNavigator } from '@react-navigation/material-bottom-tabs';
+import React from 'react';
+import MaterialCommunityIcons from 'react-native-vector-icons/MaterialCommunityIcons';
+import Friends from './Screens/Friends';
+import Home from './Screens/Home';
+import LibraryStackNavigation from './LibraryStackNavigation';
+
+const Tab = createMaterialBottomTabNavigator();
+
+//@ts-ignore
+const HomeTabs = ({navigation}): JSX.Element => {
+    return (
+        <Tab.Navigator
+            id="bottom-tab-navigator"
+            initialRouteName="home"
+            barStyle={{ backgroundColor: 'black' }}
+        >
+            <Tab.Screen
```

Comment:
In deze file kan je beter een array maken met alle tabs en die dan 1 keer loopen. Dan als je later iets wilt toevoegen hoef je dat alleen maar te doen in de array en geen html de gaan adden met dinge